### PR TITLE
Ubisys J1: Adapt exposes according to CoveringType

### PR DESCRIPTION
**Pull request by slugzero**
I have a little bit of spare time to fix some of the issues in my setup wichich are only mildly annoying: one is that the Ubisys J1 is exposing lift and tilt, even if it is configured as roller shutter with lift only.

This PR fixes this by adapting the exposes dynamically according to the `windowCoveringType`.

Note: I believe this could be useful for other devices as well since `windowCoveringType` is a standard ZCL type. Therefore it might be good to add it directly to the `cover()` expose. But I don't want to break things for other devices without testing

### CODE REVIEW

**Comment by sjorge** [comment]
Nice! This was actually on my very very long list of things I wanted to eventually tacked, every since I did dynamic expose for the SMaBiT 902010/32. Thanks for tackling this one. I can remove it from the list :)

File: src/devices/ubisys.ts [hunk 1]
```diff
@@ -815,11 +815,38 @@ const definitions: Definition[] = [
...
+                    break;
+                }
+            } else {
+                coverExpose.withPosition().withTilt();
+            }
+            return [
+                coverExpose,
+                e.power().withAccess(ea.STATE_GET),
+                e.energy().withAccess(ea.STATE_GET),
+                e.linkquality()];
```
Comment (sjorge) [inline src/devices/ubisys.ts:848]: For readability you might want to move the `]` to it's own line.

**Review by sjorge — APPROVED** [review]

File: src/devices/ubisys.ts (hunk 1 above)
Comment (sjorge) [reply src/devices/ubisys.ts]: Might need to push another lint fix as you also need a `,` after `e.linkquality()`

Comment (slugzero) [reply src/devices/ubisys.ts]: yes, just noticed that, too. thanks :-)

**Comment by Koenkk** [comment]
Cool, thanks!

**Pull request merged**